generate_config.sh

```diff
@@ -39,6 +39,20 @@ else
   read -p "Timezone: " -ei ${TZ} TZ
 fi
 
+if [ $(awk '/MemTotal/ {print $2}' /proc/meminfo) -le "1572864" ]; then  #this is 1500Mib converted into kb which /proc/meminfo reports in
+ read -r -p "The server you're using don't have enough RAM to run ClamAV in stable conditions! Do you want to disable the service? You can enable it in mailcow.conf later. [Y/n] " response
+  case $response in
+    [nN][oO]|[nN])
+      SKIP_CLAMD=n
+      ;;
+    *)
+      SKIP_CLAMD=y
+    ;;
+  esac
+else
+ SKIP_CLAMD=y
```

Comment:
Shouldn't this have been `SKIP_CLAM=n`?

Reply:
Yes you are right. I used a different approach on checking it and didn't change this one. Thanks!
Fixed in https://github.com/mailcow/mailcow-dockerized/pull/1135/commits/1741cc176ee9499c5ebbb1799b139cb81f900c1f